src/pokemon_2.c

```diff
@@ -50,12 +50,6 @@ extern u8 gUnknown_0820823C[];
 extern u8 gStatStageRatios[][2];
 extern u8 gHoldEffectToType[][2];
 
-extern u8 battle_side_get_owner(u8);
-extern u8 battle_get_side_with_given_state(u8);
-extern u32 battle_get_per_side_status(u8);
-extern u8 sub_8018324(u8, u8, u8, u8, u16);
-extern u8 sub_803C348(u8);
-
 u8 sub_803C348(u8 a1)
```

Comment:
Has two errors in the generated pokemon_2.s:

(left is `master`, right is this PR)
![image](https://cloud.githubusercontent.com/assets/737603/20754110/a763efc0-b709-11e6-8ca5-28c6951d93ed.png)
![image](https://cloud.githubusercontent.com/assets/737603/20754128/b649cae6-b709-11e6-9d50-ac5fa3dc7376.png)